node/secrets-cli/controllers/secrets.controller.js

```diff
@@ -0,0 +1,60 @@
+const db = require('../models')
+const {  generateKey, encrypt, decrypt } = require('../lib/crypto')
+
+module.exports = {
+    async createSecret(user, pass, name, value) {
+
+        const secretKey = generateKey(pass)
+        const randomKey = user.randomKey
+        const encrypted = encrypt(value, secretKey, randomKey)
+
+        return await db.Secret.create({
```

Comment:
`return await` is an anti pattern here, I'll suggest rewriting it like this: 

```suggestion
        return db.Secret.create({
```

and remove the `async` keyword since we aren't doing an await here